index.js

```diff
@@ -33,6 +33,10 @@ const handleFile = async (sourcePath, {destination, plugins = []}) => {
 		return returnValue;
 	}
 
+	if (preserveDirectories) {
+		returnValue.destinationPath = path.join(path.dirname(destinationPath), sourcePath.replace(baseDirectory, ''));
```

Comment:
I don't think using `.replace` will work cross-platform. Use the `path` related methods for path manipulation.